### PR TITLE
docs: Nov 11/15 changelog

### PR DESCRIPTION
**Pull request by danieltprice**
https://neon-next-git-dprice-changelog-11-15-2024-neondatabase.vercel.app/docs/changelog

### CODE REVIEW

File: content/changelog/2024-11-15.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  The `Updated At` value was removed from the `branches list` command output. This value reflected internal metadata changes only and provided limited value.
+
+- **Drizzle Studio update**
+
+  The Drizzle Studio integration that powers the **Tables** page in the Neon Console has been updated to version 0.0.20. For improvements and fixes in this version, see the [Neon Drizzle Studio Integration Changelog](https://github.com/neondatabase/neon-drizzle-studio-changelog/blob/main/CHANGELOG.md).
+
+- **Fixes**
+
+  - Fixed an issue where users who were removed from an organiation got an error page when logging in to Neon. The console was incorrectly redirecting them to the organization page, which they can no longer access. Users are now directed to their personal account **Projects** page instead.
```
Comment (danieltprice) [inline content/changelog/2024-11-15.md:60]:
```suggestion
  - Fixed an issue where users who were removed from an organization got an error page when logging in to Neon. The console was incorrectly redirecting them to the organization page, which they can no longer access. Users are now directed to their personal account **Projects** page instead.
```

File: content/changelog/2024-11-15.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  - When you make changes to your first or last name in **Account Settings**, those changes are now immediately reflected. Previously, old values could sometimes persist until the page was reloaded.
+
+    <Admonition type="tip" title="Did you know?">
```
Comment (danieltprice) [inline content/changelog/2024-11-15.md:63]: maybe bring this out of the list of fixes? Last week it was related to the item listed above it. wdyt?

**Review by YanicNeon — APPROVED** [review]
LGTM!

File: content/changelog/2024-11-15.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+description: Branch archiving and various improvements
```
Comment (danieltprice) [inline content/changelog/2024-11-15.md:2]:
```suggestion
description: Branch archiving, Postgres version updates, and other improvements
```

**Review by tristan957 — APPROVED** [review]

**Pull request merged**